docs/readthedocs/source/doc/Nano/Overview/troubshooting.md

```diff
@@ -3,9 +3,9 @@
 Refer to this section for common issues faced while using BigDL-Nano.
 
 ## Installation
-### Why I fail to install openvino==2022.2 when ``pip install bgdl-nano[inference]``?
+### Why I fail to install openvino-dev==2022.2 when ``pip install bigdl-nano[inference]``?
 
-Please check your system first as openvino 2022.x does not support centos anymore. Refer [OpenVINO release notes](https://www.intel.com/content/www/us/en/developer/articles/release-notes/openvino-relnotes-2021.html) for more details.
+Please check your system first as openvino-dev 2022.2 does not support centos. Refer [this](https://pypi.org/project/openvino-dev/) for more details. You can install openvino-dev==2022.3 instead as openvino-dev 2022.3 supports centos again. 
```

Comment:
we can tell user to install bigdl-nano >= 2.2 instead of installing openvino-dev 2022.3 manually

Reply:
and we may tell users 2022.3 still could not work properly on centOS :(

Reply:
Now it seems 2022.3 could work on centOS with jemalloc